ui/package.json

```diff
@@ -34,13 +34,13 @@
     "test:unit": "vue-cli-service test:unit"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.3.0",
-    "@fortawesome/free-brands-svg-icons": "^5.15.2",
-    "@fortawesome/free-solid-svg-icons": "^5.15.2",
-    "@fortawesome/vue-fontawesome": "^3.0.0-4",
+    "@fortawesome/fontawesome-svg-core": "^6.4.2",
+    "@fortawesome/free-brands-svg-icons": "^6.4.2",
+    "@fortawesome/free-solid-svg-icons": "^6.4.2",
+    "@fortawesome/vue-fontawesome": "^3.0.3",
     "@vue-js-cron/ant": "^1.1.3",
     "@vue-js-cron/core": "^3.7.1",
-    "ant-design-vue": "^3.2.9",
+    "ant-design-vue": "^3.2.20",
```

Comment:
cc @borisstoyanov @vladimirpetrov @kiranchavala tbd if full UI smoketest is required?